activiti-cloud-examples/activiti-cloud-identity-adapter/Dockerfile

```diff
@@ -1,5 +1,7 @@
-#FROM adoptopenjdk/openjdk11:jdk-11.0.2.7-alpine-slim
-FROM openjdk:11.0.1-jre-slim-stretch
+FROM docker.io/library/alpine:3.17@sha256:124c7d2707904eea7431fffe91522a01e5a861a624ee31d03372cc1d138a3126
```

Comment:
We're using `alpine:3.18@sha256:02bb6f428431fbc2809c5d1b41eab5a68350194fb508869a33cb1af4444c9b11` in other modules: should we align on the same version here too?